src/main/java/org/mqttbee/api/mqtt/mqtt5/message/disconnect/Mqtt5Disconnect.java

```diff
@@ -46,7 +46,7 @@
     Mqtt5DisconnectReasonCode getReasonCode();
 
     /**
-     * @return the optional expiry interval for the session, the client disconnects from with this DISCONNECT packet.
+     * @return the optional expiry interval for the session in seconds, the client disconnects from with this DISCONNECT packet.
```

Comment:
the optional expiry interval in seconds for the session, the ...
(as the subclause refers the session)